From foo@baz Mon Jan 26 17:24:33 PST 2009
Date: Mon, 26 Jan 2009 17:24:33 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: debugfs: fix up debugfs_create_size_t() inline function

From: Greg Kroah-Hartman <gregkh@suse.de>

If CONFIG_DEBUGFS is disabled, the fixup patch from Inaky will still
keep the debugfs.h file from compiling properly.

This should now be resolved with this patch.

Cc: Inaky Perez-Gonzalez <inaky@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/debugfs.h |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- a/include/linux/debugfs.h
+++ b/include/linux/debugfs.h
@@ -162,9 +162,10 @@ static inline struct dentry *debugfs_cre
 	return ERR_PTR(-ENODEV);
 }
 
-struct dentry *debugfs_create_size_t(const char *name, mode_t mode,
-				     struct dentry *parent,
-				     size_t *value)
+static inline struct dentry *debugfs_create_size_t(const char *name,
+						   mode_t mode,
+						   struct dentry *parent,
+						   size_t *value)
 {
 	return ERR_PTR(-ENODEV);
 }
